modules/vstudio/tests/cs2005/test_assembly_refs.lua

```diff
@@ -35,7 +35,9 @@
 	function suite.emptyGroup_onNoLinks()
 		prepare()
 		test.capture [[
-	<ItemGroup>
+	<ItemGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
+	</ItemGroup>
+	<ItemGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
```

Comment:
Wonder if tests should not have only one config by default, to avoid to duplicate each expected.

Reply:
Perhaps that would change other tests by limiting to one configuration? I understand it shows duplicated content on each `<ItemGroup>` but that is the expected behaviour with this PR.

Reply:
I meant,

- dedicated test(s) to see that multiple configurations works. (maybe with different and same content).
- previous tests with single configuration (So indeed all those tests will still be modified with the `Condition` part), but would avoid the duplicate nodes.

Not sure how it works neither, but, alternatively, could "Condition" be added only when content differs? (then previous test would works, and you can add a test with different content depending of the config).

Note: I am just a contributor, not a maintainer.